src/dc_receive_imf.rs

```diff
@@ -628,6 +645,11 @@ async fn add_parts(
         }
     }
 
+    if fetching_existing_messages && mime_parser.decrypting_failed {
+        *chat_id = ChatId::new(DC_CHAT_ID_TRASH);
+        info!(context, "We are only gathering old messages on first start. We do not want to add loads of non-decryptable messages to the chats.");
```

Comment:
This goes into the log, so this message is better as a comment. For log, could be something like `dropping existing non-decipherable message in chat ...`.